### PR TITLE
Feature/general Second milestone

### PR DESCRIPTION
**Pull request by AmineDerbal**
In this milestone :

- I added the work section.
- I added the about me section 
- implemented the interaction for the oranges buttons
- Separated the card works into 2 models. 
- the card_works ID is the grid container for the models.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,5 +59,139 @@ <h3>I'am a Software Developer</h3>
...
+          </div>
+          <div class="model_2-project">
+            <button class="orange_button">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="about_me">
+      <div id="about_me-title">
+        <h1>About me</h1>
```
Comment (ghost) [inline index.html:221]: - [ ] In your `about me` section, Kindly adjust the distance between the header and the paragraph to match the original design.

image

### Checkout the original design

image

File: index.html [hunk 2]
```diff
@@ -59,5 +59,139 @@ <h3>I'am a Software Developer</h3>
         </ul>
       </div>
     </section>
+    <section id="works_section">
+      <div id="works_title">
```
Comment (ghost) [inline index.html:63]: - [ ] I noticed your `work-section` has only two project cards. Kindly add five more to match the original dseign

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @AmineDerbal , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Professional  documentation
✅ Good Design 👍


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @AmineDerbal, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**